### PR TITLE
Remove "Non-AMD64 Architectures" section from Known Issues

### LINKED ISSUE #3447: Remove "Non-AMD64 Architectures" section from Known Issues
**What would you like to be documented**:
I would suggest removing the "Non-AMD64 Architectures" section from Known Issues. It is not accurate given that you now do have ARM pre-built images.

**Why is this needed**:
Accuracy

### PR DESCRIPTION
**Pull request by mausearce**
Fixes #3447

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test

you have to remove the reference to the issue from the commit

**Comment by aojea** [comment]
/approve

Thanks

**Comment by aojea** [comment]
/hold

I think we should be more specific, we only publish ARM, but s390x and others more esoteric are not going to be published

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ description: |-
 * [Pod Errors Due to "too many open files"](#pod-errors-due-to-too-many-open-files) (likely [inotify] limits which are not namespaced)
 * [Docker Permission Denied](#docker-permission-denied) (ensure you have permission to use docker)
 * [Windows Containers](#windows-containers) (unsupported / infeasible)
-* [Non-AMD64 Architectures](#nonamd64-architectures) (images not pre-built yet)
```
Comment (aojea) [inline site/content/docs/user/known-issues.md:36]: Non-AMD64 and ARM64

**Comment by aojea** [comment]
thinking more about this we should keep the disclaimer for those esoteric architectures, so they are aware they need to build their own images, just make more clear that is non-amd64 and non-arm64

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (stmcginnis) [reply site/content/docs/user/known-issues.md:36]: I wonder if "Unsupported Architectures" might be better than explicitly listing out the supported ones. That could keep it flexible in the unlikely event more than AMD64 and ARM64 are ever supported.

Comment (aojea) [reply site/content/docs/user/known-issues.md:36]: I like that idea

**Comment by aojea** [comment]
/hold cancel
/lgtm
/approve

Thanks

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (mausearce) [reply site/content/docs/user/known-issues.md:36]: I just pushed a new commit with this suggestion. Let's hope the 3rd time is the charm :-)

**Comment by BenTheElder** [comment]
Thanks!

**Pull request merged**